src/braket/circuits/angled_gate.py

```diff
@@ -41,22 +48,32 @@ def __init__(self, angle: float, qubit_count: Optional[int], ascii_symbols: Sequ
         super().__init__(qubit_count=qubit_count, ascii_symbols=ascii_symbols)
         if angle is None:
             raise ValueError("angle must not be None")
-        self._angle = float(angle)  # explicit casting in case angle is e.g. np.float32
+        if isinstance(angle, FreeParameter):
+            self._angle = angle
+            self._parameterized = True
```

Comment:
Should we actually need to keep track of this, or should we just return isinstance of angle in the property?

Reply:
This is used as a check across objects that also use `Parameterized`. As some parameterized objects in the future may not be `AngledGates`, there is a need for a generalized way of tracking this for different instructions.